### PR TITLE
Add .mailmap

### PR DESCRIPTION
**Pull request by wjt**
I can't remember quite why I decided to do this, but I had it lying
around.

### CODE REVIEW

File: .mailmap [hunk 1]
```diff
@@ -0,0 +1,20 @@
+Chris Chiu <chiu@endlessos.org> <chiu@endlessm.com>
+Cosimo Cecchi <cosimo@endlessm.com> <cosimoc@gnome.org>
+Dan Nicholson <dbn@endlessos.org> <nicholson@endlessm.com>
+Derek Foreman <derek@endlessm.com>
+Devin Ekins <devinj.ekins@gmail.com>
+Devin Ekins <devinj.ekins@gmail.com> <DevinJ.Ekins@gmail.com>
+Kurt von Laven <kurt@endlessm.com>
```
Comment (Kurt-von-Laven) [inline .mailmap:7]: You can map me to kurt.von.laven@gmail.com since this email no longer exists. Thanks for helping maintain this project!

**Review by Kurt-von-Laven — CHANGES_REQUESTED** [review]

File: .mailmap (hunk 1 above)
Comment (wjt) [reply .mailmap:7]: Hi, great to hear from you! I think we missed overlapping at Endless by just a couple of months.

Comment (Kurt-von-Laven) [reply .mailmap]: Bad luck!

Comment (Kurt-von-Laven) [reply .mailmap]: As a thinly veiled excuse to meet, I am happy to share everything I think I did wrong/would have done differently in retrospect If the metrics system ever undergoes a major refactor.

**Review by dbnicholson — APPROVED** [review]
I guess https://git-scm.com/docs/gitmailmap? Seems nice to have the commits in this deep history point to people as they exist now.

File: .mailmap (hunk 1 above)
Comment (wjt) [reply .mailmap]: That'd be great – I'll drop you an email!

**Pull request merged**